ts/pages/mesh/liquidity_market.tsx

```diff
@@ -0,0 +1,419 @@
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+
+import { BigNumber } from '@0x/utils';
+
+import { LiquidityMarketSelect } from 'ts/pages/mesh/liquidity_market_select';
+
+import { configs } from 'ts/utils/configs';
+import { fetchUtils } from 'ts/utils/fetch_utils';
+import { formatNumber, formatPercent } from 'ts/utils/format_number';
+import { utils } from 'ts/utils/utils';
+
+import { SignedOrder } from '@0x/types';
+
+interface OrderbookRecord {
+    order: SignedOrder;
+    metaData: {
+        orderHash: string;
+        remainingFillableTakerAssetAmount: string;
+    };
+}
+
+interface OrderData {
+    order: SignedOrder;
+    price: BigNumber;
+    size: BigNumber;
+    slippage?: BigNumber;
+}
+
+type Operation = 'ask' | 'bid';
+
+const PLACEHOLDER = '-';
+const ORDER_COUNT = 5;
+
+const orderbookBaseUrl = configs.API_BASE_PROD_URL;
+const orderbookPath = '/sra/v3/orderbook';
+
+const fetchOrders = async (baseAssetData: string, quoteAssetData: string) =>
+    fetchUtils.requestAsync(orderbookBaseUrl, orderbookPath, { baseAssetData, quoteAssetData });
+
+const markets = [
+    {
+        title: 'ETH/DAI',
+        quoteAsset: '0xf47261b00000000000000000000000006b175474e89094c44da98b954eedeac495271d0f',
+        baseAsset: '0xf47261b0000000000000000000000000c02aaa39b223fe8d0a0e5c4f27ead9083c756cc2',
+        orderSizeThreshold: 50,
+        getPrice: (order: SignedOrder, op: Operation) => {
+            let price;
+            const takerAssetAmount = new BigNumber(order.takerAssetAmount);
+            const makerAssetAmount = new BigNumber(order.makerAssetAmount);
+
+            // tslint:disable-next-line:switch-default
+            switch (op) {
+                case 'ask':
+                    price = takerAssetAmount.div(makerAssetAmount);
+                    break;
+                case 'bid':
+                    price = makerAssetAmount.div(takerAssetAmount);
+                    break;
+            }
+
+            return price;
+        },
+        getSize: (order: SignedOrder, op: Operation) => {
+            let size;
+            const takerAssetAmount = new BigNumber(order.takerAssetAmount);
+            const makerAssetAmount = new BigNumber(order.makerAssetAmount);
+            const constant = new BigNumber('1.0e18');
+
+            // tslint:disable-next-line:switch-default
+            switch (op) {
+                case 'ask':
+                    size = takerAssetAmount.div(constant);
+                    break;
+                case 'bid':
+                    size = makerAssetAmount.div(constant);
+            }
+
+            return size;
+        },
+    },
```

Comment:
would modify the function to use `remainingFillableTakerAssetAmount` instead of makerAssetAmount/takerAssetAmounts.
The function need to accept metadata (to be passed in the .map call in line 209)

I think it's best if I paste what I used locally, it seems to work better as insignificant orders get pruned.


```
getSize: (order: SignedOrder, metaData: any, op: Operation) => {
            let size;
            const takerAssetAmount = new BigNumber(order.takerAssetAmount);
            const makerAssetAmount = new BigNumber(order.makerAssetAmount);
            const remainingFillableTakerAssetAmount = new BigNumber(metaData.remainingFillableTakerAssetAmount);
            const constant = new BigNumber('1.0e18');

            // tslint:disable-next-line:switch-default
            switch (op) {
                case 'ask':
                    //size = takerAssetAmount.div(constant);
                    size = remainingFillableTakerAssetAmount.div(constant);
                    break;
                case 'bid':
                    //size = makerAssetAmount.div(constant);
                    size = remainingFillableTakerAssetAmount.div(takerAssetAmount).multipliedBy(makerAssetAmount).div(constant);
            }

            return size;
        },
```